.travis.yml

```diff
@@ -6,7 +6,7 @@ services:
   - docker
 
 env:
-  - RIOT_BRANCH=2020.10-branch
+  - RIOT_BRANCH=2021.01-branch
```

Comment:
this should IMO be a distinct PR (and creating it should be part of the release process)

Reply:
Here it is: #133